packages/protocol/test/Vault.t.sol

```diff
@@ -281,40 +411,65 @@ contract VaultTest is DSTestPlus {
     assertEq(vault.balanceOfDebt(bob), 0);
   }
 
-  function test_liquidateDefault() public {
+  function test_liquidateDefault(uint256 priceDrop) public {
     uint256 amount = 1 ether;
     uint256 borrowAmount = 1000e18;
+
+    vm.assume(
+      priceDrop
+        > _utils_getLiquidationThresholdValue(
+          address(vault), TEST_USD_PER_ETH_PRICE, amount, borrowAmount
+        )
+    );
+
+    uint256 price = oracle.getPriceOf(address(debtAsset), address(asset), 18);
+    uint256 liqRatio = IBVaultTestingExtOnly(address(vault)).liqRatio();
+    uint256 priceDropThresholdToMaxLiq = price - ((95 * borrowAmount * 1e34) / (amount * liqRatio));
```

Comment:
it's not clear to me how we got to `1e34`, maybe assign this value to a const whose name hints at its meaning or its derivation

Reply:
1e34 is because the 95 is actually 95e16 and borrowAmount is 1e18. Changed it to: 95e16 * borrowAmount * 1e18